### PR TITLE
Do not marshal/unmarshal to ByteBufs during 'local' communication  (part 2)

### PR DESCRIPTION
**Pull request by eolivelli**
- Implement executeUpdates
- add more tests

### CODE REVIEW

**Comment by eolivelli** [comment]
In this patch I am addressing @nicoloboschi 's comments here
https://github.com/diennea/herddb/pull/725

File: herddb-core/src/main/java/herddb/server/ServerSideConnectionPeer.java [hunk 1]
```diff
@@ -1180,4 +1182,115 @@ public LocalClientScanResultSetImpl executeScan(String tableSpace, String query,
...
+                List<Object> parameters = batch.get(i);
+                TranslatedQuery translatedQuery = server
+                        .getManager()
+                        .getPlanner().translate(tableSpace, query,
+                                parameters, false, true, returnValues, -1);
+                queries.add(translatedQuery);
+            }
+
+            List<Long> updateCounts = new CopyOnWriteArrayList<>();
+            List<Map<RawString, Object>> otherDatas = new CopyOnWriteArrayList<>();
```
Comment (nicoloboschi) [inline herddb-core/src/main/java/herddb/server/ServerSideConnectionPeer.java:1207]: typo "otherData"

**Review by nicoloboschi — COMMENTED** [review]
LGTM, there is a typo 
I'm fine if you want to merge anyway

File: herddb-core/src/main/java/herddb/server/ServerSideConnectionPeer.java (hunk 1 above)
Comment (eolivelli) [reply herddb-core/src/main/java/herddb/server/ServerSideConnectionPeer.java:1207]: this is not a typo, this is a list of "otherData", so "otherDatas" sounds good to me, do you have a suggestion ?

**Comment by eolivelli** [comment]
tests passed, Travis broke on Coveralls submission
https://travis-ci.org/github/diennea/herddb/builds/739290165

**Pull request merged**